hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -20,7 +20,7 @@
 <jboss-deployment-structure>
   <deployment>
     <dependencies>
-      <module name="org.hawkular.bus" />
+      <module name="deployment.hawkular-alerts-rest.war" />
```

Comment:
Will there be any problem if alerting WAR is deployed inside of a metrics EAR, and not independently? Perhaps we need to to add  optional="true", and potentially add the other deployment scenario?